randovania/games/prime1/layout/preset_describer.py

```diff
@@ -17,8 +17,8 @@
     LayoutCutsceneMode.MAJOR: "Major cutscene removal",
     LayoutCutsceneMode.MINOR: "Minor cutscene removal",
     LayoutCutsceneMode.COMPETITIVE: "Competitive cutscene removal",
-    LayoutCutsceneMode.SKIPPABLE: "Skippable cutscenes (Experimental)",
-    LayoutCutsceneMode.SKIPPABLE_COMPETITIVE: "Competitive cutscene removal (Experimental)",
+    LayoutCutsceneMode.SKIPPABLE: "Skippable cutscenes",
```

Comment:
We could go with skippable cutscenes not being mentioned, but vanilla mentions they're not.